udp-ballerina/build.gradle

```diff
@@ -203,7 +206,7 @@ task ballerinaBuild {
             if (Os.isFamily(Os.FAMILY_WINDOWS)) {
                 commandLine 'cmd', '/c', "$balJavaDebugParam $distributionBinPath/bal.bat build -c ${testParams} ${debugParams} && exit %%ERRORLEVEL%%"
             } else {
-                commandLine 'sh', '-c', "$balJavaDebugParam $distributionBinPath/bal build -c ${testParams} ${debugParams}"
+                commandLine 'sh', '-c', "$balJavaDebugParam $distributionBinPath/bal build -c ${testParams} ${debugParams} && $distributionBinPath/bal push --repository=local"
```

Comment:
Was this change intentional?

Reply:
This is added to test the implementation in the local machine, ```&& $distributionBinPath/bal push --repository=local``` can be removed

Reply:
If it is added to test the implementation in the local machine, this shouldn't be here.

Reply:
+1, fixed in #96

Reply:
Merged it.